src/tv2-common/content/server.ts

```diff
@@ -84,7 +84,8 @@ function GetServerTimeline(
 			seek: contentProps.seek,
 			length: contentProps.seek ? contentProps.clipDuration : undefined,
 			inPoint: contentProps.seek ? 0 : undefined,
-			playing: true
+			playing: true,
+			noStarttime: true
```

Comment:
I know that it is not something that you named, but shouldn't it be `noStartTime`?